scipy/integrate/_quadpack_py.py

```diff
@@ -615,11 +617,32 @@ def dblquad(func, a, b, gfun, hfun, args=(), epsabs=1.49e-8, epsrel=1.49e-8):
         an accuracy of ``abs(i-result) <= max(epsabs, epsrel*abs(i))``
         where ``i`` = inner integral of ``func(y, x)`` from ``gfun(x)``
         to ``hfun(x)``, and ``result`` is the numerical approximation.
-        See `epsrel` below.
+        See `epsrel` below. This arugment will be deprecated in a
+        future release, instead use the `opts` argument.
     epsrel : float, optional
         Relative tolerance of the inner 1-D integrals. Default is 1.49e-8.
         If ``epsabs <= 0``, `epsrel` must be greater than both 5e-29
-        and ``50 * (machine epsilon)``. See `epsabs` above.
+        and ``50 * (machine epsilon)``. See `epsabs` above. This arugment will
+        be deprecated in a future release, instead use the `opts` argument.
+    opts : iterable object or dict, optional
+        Options to be passed to `quad`. May be empty, a dict, or
+        a sequence of dicts or functions that return a dict. If empty, the
+        default options from scipy.integrate.quad are used. If a dict, the same
+        options are used for all levels of integraion. If a sequence, then each
+        element of the sequence corresponds to a particular integration. e.g.,
+        ``opts[0]`` corresponds to integration over ``x0``, and so on. If a
+        callable, the signature must be the same as for ``ranges``. The
+        available options together with their default values are:
+
+          - epsabs = 1.49e-08
```

Comment:
As-is, this will render as a quotation; please remove the indent.

Reply:
This has been fixed and is ready. I wonder why it doesn't do this for `nquad`, perhaps becuase it's using raw strings?